doc/apiblueprints/elektrad.apib

```diff
@@ -0,0 +1,75 @@
+FORMAT: 1A
+
+# elektrad API
+
+to access single instances, each elektra daemon (`elektrad`) provides a REST
+HTTP API
+
+
+## versions [GET /version]
+
+returns the current version of the API and elektra
+
+The API version is increased whenever breaking changes
+(i.e. changes that prevent backward compatibility) are made.
+The Elektra version is directly taken from the Elektra library,
+for further information and explanation see [doc/VERSION.md](https://github.com/ElektraInitiative/libelektra/blob/master/doc/VERSION.md).
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + elektra (object) - Detailed version information about the used Elektra library
+            + version (string) - The currently used version in its complete format
+            + major (number) - The currently used major version
+            + minor (number) - The currently used minor version
+            + micro (number) - The currently used micro version
+        + api (number) - The version of the API itself
+
+
+## elektra key database [/kdb/{+path}]
+
+access the elektra key database by specifying a `path`
+
++ Parameters
+    + path: `user/hello` (string) - path to the elektra config
+
+### get configuration [GET]
+
+this is the same as calling `kdb get {path}`
```

Comment:
Is this only the split-up or did you already update these parts? You should describe the behaviour with text or by means of C-API, not by referring to other implementations.

And we should not repeat problems already known: e.g. #401